api/v1alpha1/clusterconfig_types.go

```diff
@@ -395,8 +395,15 @@ func (User) VariableSchema() clusterv1.VariableSchema {
 					Type:        "string",
 				},
 				"hashedPassword": {
-					Description: "The hashed password for the user",
+					Description: "The hashed password for the user. Must be in the format of some hash function supported by the OS.",
 					Type:        "string",
+					// Although the crypt (5) man page lists regexes for
+					// supported hash functions. Although we could validate
+					// input against a set of regexes, this set may be different
+					// from the set supported by the chosen OS, and we could
+					// return a false negative or positive. For this reason, we
+					// do not validate the input.
+					//
```

Comment:
Were the  2 "Although" intentional?
Also "//" at the end

Reply:
Nope, typo. Fixed it. Thanks!